packages/next-on-pages/tests/src/buildApplication/processVercelFunctions/invalidFunctions.test.ts

```diff
@@ -0,0 +1,116 @@
+import { describe, test, expect, afterEach, vi } from 'vitest';
+import mockFs from 'mock-fs';
+import {
+	collectFunctionsFrom,
+	mockConsole,
+	edgeFuncDir,
+	nodejsFuncDir,
+	getRouteInfo,
+} from '../../../_helpers';
+import { resolve } from 'path';
+import { processEdgeFunctions } from '../../../../src/buildApplication/processVercelFunctions/edgeFunctions';
+import { checkInvalidFunctions } from '../../../../src/buildApplication/processVercelFunctions/invalidFunctions';
+
+const functionsDir = resolve('.vercel/output/functions');
+
+describe('checkInvalidFunctions', () => {
+	afterEach(() => mockFs.restore());
+
+	test('should ignore i18n index with valid alternative', async () => {
```

Comment:
shouldn't this and the following test also check the `ignoredFunctions` map?

Reply:
yeah sure if ya want, will add that a little later on